index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav class="menu">
+                <h4 class="logo">My Logo</h4>        
+                <img src="Icon - Menu.png" alt="Icon-menu"></nav>
+        </header>
+        
+    
+        <main class="about">
+            <section class="intro">
+                <h2>I’m Karthika. <br>
+                    Glad to see you!</h2>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+                     If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </section>
+            <section class="connect">
+                <h4>LET'S CONNECT</h4>
+            </section>
+            <section class="social">
```

Comment:
- [ ] please ensure you are using the appropriate `font-weight, color` provided for you in the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=Di4jnFTf3W7Va5nw-0) to display the texts for the title and info section. It is important to use the correct fonts which can easily be gotten from figma :+1: 
- figma font
![font](https://github.com/Porkalaikarthika/Portfolio/assets/89055502/a7d2e182-174f-4755-b4bd-7a1164910f65)